lms/templates/admin/instance.html.jinja2

```diff
@@ -80,6 +80,13 @@
         <legend class="label has-text-centered">Canvas settings</legend>
         {{ macros.disabled_text_field("API domain", instance.custom_canvas_api_domain) }}
         {{ macros.disabled_text_field("Developer key", instance.developer_key) }}
+        {{ macros.form_text_field(request, "Developer key", "developer_key", field_value=instance.developer_key) }}
+        {{ macros.form_text_field(
+            request,
+            "Developer secret",
+            "developer_secret",
+            placeholder="*" * 25 if instance.developer_secret else "")
```

Comment:
I think this is good compromise between having to decrypt the actual key and not having any hint about the key having a value currently or not.